### DISCUSSION TITLE
[Motherboard - add motherboard to inventory via GLPI-Agent]

### DISCUSSION THREAD

**Discussion by tristangallet** [glpi-project/glpi-agent#553] [Ideas]
Hello, could you add the motherboard in the inventory ? We have switched from ocsng client and in OCS we have some informations about the motherboard that we can not find in GLPI (motherboard serial for example). Thanks.

**Comment by g-bougard** [comment]
Hi @tristangallet 

have you check the inventory for this missing information ?

If it's in the inventory but not in GLPI, this means it is not exposed in GLPI. Also maybe the agent is seeing motherboard and computer serial and only expose one. In that case, the problem is in GLPI, not in GLPI-Agent.
If it's NOT in the inventory, can you share the inventory or at least the result of `glpi-inventory --partial bios` ?
